backtrace-library/src/androidTest/java/backtraceio/library/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="backtraceio.library">
```

Comment:
why such change

Reply:
This is required by gradle upgrade to version 8.2.0